x-pack/test/functional/apps/transform/creation/index_pattern/creation_index_pattern.ts

```diff
@@ -561,6 +561,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
           await transform.testExecution.logTestStep('again displays an empty index preview');
           await transform.wizard.assertIndexPreviewEmpty();
+          await transform.datePicker.assertDataTierOptionsVisible(true);
```

Comment:
Could this check also be added to the index data visualizer functional test? For example somewhere near https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer.ts#L45?

Reply:
Added here https://github.com/elastic/kibana/pull/181193/commits/761c030eddd69a8bb1e80be27c4e25b9026768bf